csrankings-b.csv

```diff
@@ -173,6 +173,7 @@ Bastian Goldlücke,University of Konstanz,https://www.cvia.uni-konstanz.de/perso
 Bastian Leibe,RWTH Aachen University,http://www.vision.rwth-aachen.de/person/1/,ZcULDB0AAAAJ
 Bastian Rieck,University of Fribourg,https://bastian.rieck.me,La7zuKQAAAAJ
 Bastin Tony Roy Savarimuthu,University of Otago,https://www.otago.ac.nz/school-of-computing/our-people/tony-savarimuthu,0__bwK0AAAAJ
+Batya Kenig,Technion,https://sites.google.com/view/batyakenig/home,nPuXokoAAAAJ

```

Comment:
Please justify inclusion (non CS). You may have done this in the past but you need to do this any time this happens. Thanks.

Reply:
Batya Kenig is a researcher in database systems. She publishes exclusively in CS conferences: ICDT, PODS, UAI, SIGMOD, AAMAS, AAAI. Her area is quite close to that of Benny Kimelfeld, who is faculty at the CS department.